### PR TITLE
Delete common

### PR DESCRIPTION
**Pull request by JoshMarangoni**
Deleted common, changed paths, instantiate opencv net in init function instead of process, other minor bug fixes

### CODE REVIEW

File: source/panel_predictor/__main__.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def run_live(framework):
 
         if cv2.waitKey(1) & 0xFF == ord('q'):  # press q to quit
             break
-        ret, frame = cap.read()  # get next frame
+        frame = cap.read()  # get next frame
```
Comment (jamesrosstwo) [inline source/panel_predictor/__main__.py:46]: Does removing `ret` work? It's my understanding from [this documentation](https://docs.opencv.org/3.4/d8/dfe/classcv_1_1VideoCapture.html#a473055e77dd7faa4d26d686226b292c1) that cap.read() will return a tuple with two values

File: source/panel_predictor/panel_finder/panel_classifier/__main__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from .panel_classifier import PanelClassifier
+from panel_classifier import PanelClassifier
```
Comment (jamesrosstwo) [inline source/panel_predictor/panel_finder/panel_classifier/__main__.py]: I think the relative import is important here, as `panel_classifier` isn't in the root directory

**Review by jamesrosstwo — CHANGES_REQUESTED** [review]
Other than two minor things, this looks ready to merge :)

File: source/panel_predictor/panel_finder/panel_classifier/__main__.py (hunk 2 above)
Comment (pedramamani) [inline source/panel_predictor/panel_finder/panel_classifier/__main__.py]: Why not importing from `source` directory?

File: source/panel_predictor/panel_finder/panel_classifier/__main__.py [hunk 3]
```diff
@@ -9,9 +9,9 @@
     parser.add_argument('-d', '--data',
                         help="Specifies the path to the data file to train the model")
     parser.add_argument('-m', '--mode', required=True,
-                        help="Sets the mode of the classifier. Options: train, load")
+                        help="Sets the mode of the classifier. Options: train, load, convert")
     parser.add_argument('-M', '--model',
-                        help="Specifies the path to the data file to store or load the trained model, (load, train, convert, pi)")
+                        help="Specifies the path to the data file to store or load the trained model, (load, train)")
```
Comment (pedramamani) [inline source/panel_predictor/panel_finder/panel_classifier/__main__.py]: Follow the same standard as line 12 for Options.

File: source/panel_predictor/panel_finder/panel_classifier/inference_opencv.py [hunk 4]
```diff
@@ -26,6 +26,7 @@ class OpenCVClassifier(Module):
     def __init__(self, parent, state=None):
         self.working_dir = Path(os.path.dirname(os.path.abspath(__file__)))
         super().__init__(self.working_dir, parent=parent, state=state)
+        self.net = self.load_model()
```
Comment (pedramamani) [inline source/panel_predictor/panel_finder/panel_classifier/inference_opencv.py]: `self.model` is more clear.

File: source/panel_predictor/panel_finder/panel_classifier/panel_classifier.py [hunk 5]
```diff
@@ -29,7 +29,7 @@ def find_ratio(a, b):
 
 
 class PanelClassifier(Module):
-    def __init__(self, parent, state=None):
+    def __init__(self, parent=None, state=None):
```
Comment (pedramamani) [inline source/panel_predictor/panel_finder/panel_classifier/panel_classifier.py:32]: This is a more general comment. Why is `parent` not a required field in `Module` @jamesrosstwo? I think it should be.

**Review by pedramamani — CHANGES_REQUESTED** [review]
Some suggestions, and some points of clarification for me.

File: source/panel_predictor/__main__.py (hunk 1 above)
Comment (cmcahoon01) [reply source/panel_predictor/__main__.py:46]: cap.read() will return a tuple when used on a video stream. unfortunately, it only returns 1 from a live input. This was previously throwing an error

Comment (JoshMarangoni) [reply source/panel_predictor/__main__.py:46]: Yes it runs, and we are also not using VideoCapture() in this function, we are using VideoStream(), which only has one return value https://github.com/jrosebr1/imutils/blob/master/imutils/video/videostream.py

File: source/panel_predictor/panel_finder/panel_classifier/__main__.py (hunk 2 above)
Comment (cmcahoon01) [reply source/panel_predictor/panel_finder/panel_classifier/__main__.py]: does this work for you? I received an error on this

**Review by cmcahoon01 — APPROVED** [review]

File: source/panel_predictor/panel_finder/panel_classifier/__main__.py (hunk 2 above)
Comment (JoshMarangoni) [reply source/panel_predictor/panel_finder/panel_classifier/__main__.py]: Right, forgot about the Pycharm folks

Comment (JoshMarangoni) [reply source/panel_predictor/panel_finder/panel_classifier/__main__.py]: Yea I guess absolute is more proper, I'll change it

File: source/panel_predictor/panel_finder/panel_classifier/panel_classifier.py (hunk 5 above)
Comment (JoshMarangoni) [reply source/panel_predictor/panel_finder/panel_classifier/panel_classifier.py:32]: It doesn't change anything if you pass in a value, but if you run panel_classifier on its own, it does not have a value for parent so it throws an error

File: source/panel_predictor/panel_finder/panel_classifier/__main__.py (hunk 2 above)
Comment (jamesrosstwo) [reply source/panel_predictor/panel_finder/panel_classifier/__main__.py]: Keep in mind that it's not pycharm that would make this work - it's the way your python environment is set up.

File: source/panel_predictor/panel_finder/panel_classifier/panel_classifier.py (hunk 5 above)
Comment (jamesrosstwo) [reply source/panel_predictor/panel_finder/panel_classifier/panel_classifier.py:32]: Maybe a better solution would be to make `parent` a require property, but when creating modules in a `__main__` file, just pass in `None`.

Comment (pedramamani) [reply source/panel_predictor/panel_finder/panel_classifier/panel_classifier.py:32]: I agree that's a better option. For another PR though.

**Review by pedramamani — APPROVED** [review]
Looks good.

**Review by jamesrosstwo — APPROVED** [review]

**Pull request merged**